app/models/user_permission.rb

```diff
@@ -0,0 +1,13 @@
+class UserPermission < ActiveRecord::Base
+  belongs_to :user
+  validates_presence_of :user_id, :resource_type
+
+  def access_rules
+    rules = super
+    rules && HashWithIndifferentAccess.new(JSON.parse(rules))
+  end
+
+  def resource_type
+    super&.constantize
```

Comment:
works for now...although we might have to change later to accommodate the legacy symfony permissions...

Reply:
+1 (but postponing for now...)